server/config/validate-config-keys.js

```diff
@@ -46,15 +46,16 @@ const sharedConfig = parseConfig( '_shared.json' );
  * invalid. Such a missing value could and likely would
  * cause runtime errors in Calypso
  */
-environmentKeys.forEach( ( [ filename, keys ] ) => {
+environmentKeys.forEach( ( params ) => {
+	const keys = params[ 1 ];
 	keys.forEach( key => {
 		if ( ! sharedConfig.hasOwnProperty( key ) ) {
 			console.error(
 				`${ chalk.red( 'Configuration Error' ) }\n` +
 				`Key ${ chalk.blue( key ) } defined in ${ chalk.blue( filename ) } ` +
 				`but not in ${ chalk.blue( '_shared.json' ) }\n` +
 				`Please add a default value in ${ chalk.blue( '_shared.json' ) } ` +
-				`before adding overrides in the environment-specific config files.`
+				'before adding overrides in the environment-specific config files.'
```

Comment:
Are these changes accomplishing any goals? It seems like they don't actually change anything, or was the list destructuring a problem with node?